test/sequential/test-fs-write-buffer-without-length.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const path = require('path');
+const Buffer = require('buffer').Buffer;
+const fs = require('fs');
+const filename = path.join(common.tmpDir, 'write.txt');
+
+common.refreshTmpDir();
+
+fs.open(filename, 'w', 0o644, common.mustCall(function(err, fd) {
+  if (err) throw err;
+
+  fs.write(fd,
+           Buffer.from('hello'),
+           3,
+           common.mustCall(function(err, written) {
```

Comment:
This looks a bit hectic. What about moving this `common.mustCall(...)` to a variable and passing that as the fourth argument instead? That way the `fs.write()` fits all on one line.